src/Plugin_Command.php

```diff
@@ -375,6 +379,9 @@ public function activate( $args, $assoc_args = array() ) {
 	 * [--all]
 	 * : If set, all plugins will be deactivated.
 	 *
+	 *  [--exclude=<name>]
+	 * : Comma separated list of plugin names that should be excluded from deactivation.
```

Comment:
```suggestion
	 * : Comma separated list of plugin slugs that should be excluded from deactivation.
```

For consistency. It says slugs for the other two commands as well.